File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/PatchApplicationVersionExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class PatchApplicationVersionExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        var patchOperation1 = new PatchOperation(
            op: "replace",
            path: "/supported"
        );

        var patchOperation = new List<PatchOperation>
        {
            patchOperation1,
        };

        try
        {
            var response = new ApplicationsBetaApi(config).PatchApplicationVersion(
                applicationKey: "applicationKey_string",
                versionKey: "versionKey_string",
                patchOperation: patchOperation
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ApplicationsBetaApi#PatchApplicationVersion: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/PatchApplicationVersionExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const patchOperation1: api.PatchOperation = {
  op: "replace",
  path: "/supported",
};

const patchOperation = [
  patchOperation1,
];

new api.ApplicationsBetaApi(configuration).patchApplicationVersion(
  "applicationKey_string", // applicationKey
  "versionKey_string", // versionKey
  patchOperation,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ApplicationsBetaApi#patchApplicationVersion:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
